### ISSUE TITLE
Ability to edit points in FitFileExplorer

### ISSUE THREAD

**Issue by DanHubb** [roznet/connectstats#69]
Unlike other programs, FitFileExplorer can show the rest of an activity after a bad point. It would be nice to be able to delete the bad data points from the fit file and save the edited file.

**Comment by roznet** [comment]
Yes, this is a good idea I have been thinking about. It's a bit involved but will work on this. It may take a bit of time though.

**Comment by lukas-mertens** [comment]
@roznet This exactly the reason why I installed the program. I would like to edit my .fit-file to add a distance to my apple watch recording for indoor cycling.
